cli/userInfo.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/hashicorp/vault/api"
+)
+
+// Connects to Vault server
+func Client(token string) (*api.Client, error) {
+	config := api.DefaultConfig()
+
+	config.Address = "http://127.0.0.1:8200"
```

Comment:
This should be configurable. Set this in a separate file akin to https://github.com/CanDIG/GraphQL-interface/blob/master/api/settings.py , then call it as a constant.